terminus-ui/file-upload/src/selected-file.spec.ts

```diff
@@ -33,8 +32,8 @@ const FILE_BLOB = new Blob(
   ['data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAEgAAABIAQMAAABvIyEEAAAAA1BMVEXXbFn0Q9OUAAAADklEQVR4AWMYRmAUjAIAAtAAAaW+yXMAAAAASUVORK5CYII='],
   { type: 'image/png' },
 );
-FILE_BLOB.lastModifiedDate = new Date();
-FILE_BLOB.name = 'foo';
+FILE_BLOB['lastModifiedDate'] = new Date();
+FILE_BLOB['name'] = 'foo';
```

Comment:
I'm surprised that lint isn't yelling at you for not using dot notation..

Reply:
It was actually yelling at me for **using** dot notation. It doesn't think those items exist. Probably a typing issue, but it's just a mock blob for tests so I didn't spend much time on it.

Reply:
interesting.... i found a few times lint secretly "fixed" code by converting to dot notation and broke functionality.

Reply:
Yeah it definitely prefers it since we lose typings using the index format. I wouldn't have done this in production code 😬

Reply:
ah just realized it's specs(you did mention it last comment but that flew by for some reason)... never mind..